source/installation.txt

```diff
@@ -90,25 +90,33 @@ command:
 Install from Source
 ~~~~~~~~~~~~~~~~~~~
 
-If the above options still do not allow you to install ``pymongoarrow`` on your
+If you can't use the above options to install ``pymongoarrow`` on your
 system, you can install from source. To learn how, see the `Contributing Guide
 <https://github.com/mongodb-labs/mongo-arrow/blob/main/bindings/python/CONTRIBUTING.md#installing-from-source>`__.
 
 Dependencies
 ^^^^^^^^^^^^
 
-{+driver-short+} requires the following:
+{+driver-short+} requires the following dependencies:
 
-- PyMongo>=4.4
-- PyArrow>=16,<16.1
+- PyMongo version 4.4 or later
+- PyArrow version 16
```

Comment:
I: The PyMongoArrow code explicitly requires PyArrow<16.1, so that requirement is probably worth keeping

```suggestion
- PyArrow version 16.0.x
```

Reply:
I specifically put PyArrow version 16 because at least based on my research, the only version of [PyArrow between 16 and 16.1 is 16](https://arrow.apache.org/release/), there's no 16.0.x versions. Do you think adding 16.0.x would be more clear? I was concerned that it would imply that they should use a version that no longer exists, since it seems like the only version compatible is specifically version 16.

Reply:
How about "16.0"? That would clarify that it's not compatible with 16.1.